src/pages/DataSetReport.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Paper } from 'material-ui'
+import manageError from '../utils/pageEnhancers/manageError.HOC'
+import Feedback from '../components/Feedback'
+import { Headline } from './data-set-report/Headline'
+import { DataInputs } from './data-set-report/DataInputs'
+import { FormActions } from './data-set-report/FormActions'
+import { HtmlReport } from './data-set-report/HtmlReport'
+import { connectDataSetReport } from './data-set-report/connectDataSetReport'
+import { isActionEnabled } from './data-set-report/helpers'
+import styles from '../utils/styles'
+
+class DataSetReport extends React.Component {
```

Comment:
Unfortunately that's impossible because we need the higher order component to be able to extend the class right now. We might find a better solution once everything is refactored though. I'll think about a proper solution for this.

Reply:
Yeah, perhaps something like this: https://reactjs.org/docs/error-boundaries.html. No action needed now, we'll fix it later.